packages/legend-application-studio/src/stores/EditorStore.ts

```diff
@@ -1187,11 +932,100 @@ export class EditorStore implements CommandRegistrar {
     );
   }
 
-  changeLocalChangesState(): void {
-    if (this.isInFormMode) {
-      this.localChangesState = new FormLocalChangesState(this, this.sdlcState);
-    } else {
-      this.localChangesState = new TextLocalChangesState(this, this.sdlcState);
+  *switchModes(
```

Comment:
What do we decide to do with this function, or are you still in the midst of addressing the comments?

Reply:
probably can move these 2 methods to inside of `EditorGraphState` now